<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerWin</class>
 <widget class="QDialog" name="customerWin">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>customerWin</string>
  </property>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Customer's Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>example@example.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="emailLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>E-mail</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>(123)456-7890</string>
   </property>
  </widget>
  <widget class="QLabel" name="phoneLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone</string>
   </property>
  </widget>
  <widget class="QLabel" name="addressLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Acme Street, 99</string>
   </property>
  </widget>
  <widget class="QLabel" name="alreadyRegLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>nameLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>emailLineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>emailLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>phoneLineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>phoneLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addressLineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addressLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>saveButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
